setup.cfg

```diff
@@ -13,10 +13,39 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-  prompt_toolkit==1.0.15
-  whaaaaat
   jinja2
-  pyyaml
+  click==6.7
+  easydict==1.6
+  matplotlib==2.2.2
+  numpy==1.17.4
+  pandas==0.19.2
+  Pillow==6.2.1
+  prompt_toolkit==1.0.15
+  pytablewriter==0.47.0
+  PyYAML==4.2b4
+  tensorflow-datasets==1.0.2
+  whaaaaat==0.5.2
+  cython==0.27.3
+
+[options.extras_require]
+cpu =
+  tensorflow==1.15.2
+gpu =
+  tensorflow-gpu==1.15.2
+  gast==0.2.2
```

Comment:
nits: `gast` looks just the dependency of tensorflow

Reply:
@kchygoe Wow, then we can remove this?